### DISCUSSION TITLE
How to multi line text without Enter click?

### DISCUSSION THREAD

**Discussion by Matios210** [seleniumbase/SeleniumBase#2444] [Q&A] [answered]
So my question is how can i type in multi lines into text box without seleniumbase clicking Enter?
Something like Shift + Enter to get into new line to not trigger "Send" button

**Comment by mdmintz** [comment, marked as answer]
As long as your text doesn't end with `\n`, then when you type text into a text field, such as with `self.type(selector, text)`, then it won't call `self.submit(selector)` on that element, which triggers a Send, etc.
